src/Runner.Common/Constants.cs

```diff
@@ -155,6 +155,7 @@ public static class ReturnCode
                 public const int RunnerUpdating = 3;
                 public const int RunOnceRunnerUpdating = 4;
                 public const int SessionConflict = 5;
+                public const int RunnerConfigurationRefreshed = 6;
```

Comment:
can we add some comment to indicates we can remove this return code later, and the service script will never receive this return code?